### DISCUSSION TITLE
Github larger runner takes long to start

### DISCUSSION THREAD

**Discussion by BioCarmen** [community/community#51970] [Actions]
### Select Topic Area

Question

### Body

I sometimes have to wait a long time for the 4 cores larger runners to start. I thought that because we hit a limit but based on the documentation it should support 500 total concurrent jobs and we are not running that many for sure. 
Anyone has insight on what might be happening? And is there a way to introduce a fallback for this? (if it doesn't start in 10 minutes, we should just switch to use the standard runner?
```
Requested labels: ubuntu-latest-4-cores
Job defined at: Myorg/myrepo/.github/workflows/my-workflow.yml@refs/pull/11982/merge
Waiting for a runner to pick up this job...
```

**Comment by Akash1134** [comment]
Hi @BioCarmen ,

Have you checked the workflow for the `runs-on` definition? The Required labels might have a typo as discussed in[ this discussion](https://github.com/orgs/community/discussions/31587). 

If the workflow file is correct or you still have the same issue after fixing it, the best option is probably to [submit a bug report](https://support.github.com/contact/bug-report).   

Thanks!

**Comment by BioCarmen** [comment]
Thanks @Akash1134 i don't think it is the runs-on definition because it runs well most of the time. I will create a bug report!
